portal/views/portal.py

```diff
@@ -1036,8 +1039,20 @@ def get_reporting_counts():
             counts['interventions'][interv.description] += 1
             if (any(doc.intervention == interv for doc in user.documents)):
                 counts['intervention_reports'][interv.description] += 1
-        for org in user.organizations:
-            counts['organizations'][org.name] += 1
+        if not user.organizations:
+            counts['organizations']['Unspecified'] += 1
+        else:
+            for org in user.organizations:
+                counts['organizations'][org.name] += 1
+        counts['registrations'].append(user.registered)
+
+    for enc in Encounter.query.filter_by(auth_method='password_authenticated',
+                                         status='finished'):
```

Comment:
FWIW, we have a bug around `finished` encounters, which will get cleaned up when Ivan's [Session Monitor](https://github.com/uwcirg/true_nth_usa_portal/pull/804) solution is ready, that is, encounters only get marked finished if the browser session isn't killed.

You might ignore the `status` for now.